nautobot/core/api/urls.py

```diff
@@ -36,4 +38,7 @@
     path("graphql/", GraphQLDRFAPIView.as_view(), name="graphql-api"),
     # Plugins
     path("plugins/", include((plugin_api_patterns, "plugins-api"))),
+    # Lookup Expr
+    path("lookup-choices/", GetFilterSetFieldLookupExpressionChoicesAPI.as_view(), name="lookup_choices"),
+    path("lookup-value-dom-element/", GetFilterSetFieldDOMElementAPI.as_view(), name="lookup_value_dom_element"),
```

Comment:
Perhaps something like:

```suggestion
    path("filterset-fields/lookup-choices/", GetFilterSetFieldLookupExpressionChoicesAPI.as_view(), name="lookup_choices"),
    path("filterset-fields/lookup-value-dom-element/", GetFilterSetFieldDOMElementAPI.as_view(), name="lookup_value_dom_element"),
```